<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BalansSets</class>
 <widget class="QWidget" name="BalansSets">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>591</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="mainLayout">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <item>
     <layout class="QGridLayout" name="gridLayout" columnstretch="0,0">
      <property name="margin">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Наименование:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="txtName">
          <property name="minimumSize">
           <size>
            <width>268</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="topMargin">
         <number>5</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="cmdSave">
          <property name="text">
           <string>Сохранить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cmdCancel">
          <property name="text">
           <string>Сбросить</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Элементы:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTableWidget" name="twElements">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <attribute name="verticalHeaderCascadingSectionResizes">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderDefaultSectionSize">
             <number>20</number>
            </attribute>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="leftMargin">
             <number>5</number>
            </property>
            <item>
             <widget class="QPushButton" name="cmdAdd">
              <property name="text">
               <string>Добавить</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="cmdRemove">
              <property name="text">
               <string>Удалить</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="cmdRemoveBalans">
            <property name="text">
             <string>Удалить балансовую группу</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Действия:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cmdAdd</sender>
   <signal>clicked()</signal>
   <receiver>BalansSets</receiver>
   <slot>addChannels()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>594</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>513</x>
     <y>790</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdSave</sender>
   <signal>clicked()</signal>
   <receiver>BalansSets</receiver>
   <slot>saveChanges()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>503</x>
     <y>566</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>768</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdCancel</sender>
   <signal>clicked()</signal>
   <receiver>BalansSets</receiver>
   <slot>dropChanges()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>567</x>
     <y>573</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>796</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdRemove</sender>
   <signal>clicked()</signal>
   <receiver>BalansSets</receiver>
   <slot>removeChannels()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>555</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>801</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addChannels()</slot>
  <slot>saveChanges()</slot>
  <slot>dropChanges()</slot>
  <slot>removeChannels()</slot>
 </slots>
</ui>
